mitodl/micromasters
# Installed celery

#### What are the relevant tickets?
Fixes #113 

#### What's this PR do?
Installs celery so we can use it for updating enrollment data later (#53) .

#### Where should the reviewer start?
micromasters/celery.py

#### How should this be manually tested?
Go to /status/?token=... and verify that the celery setting exists and it's up and running

#### Any background context you want to provide?

#### Screenshots (if appropriate)

#### What GIF best describes this PR or how it makes you feel?
http://i.imgur.com/xdPg9kp.gifv
